[Suggestion] Only use GPU if it's available

Hi!

I don't have a GPU, and when I try to initialise the `PunctFixer` as follows I get an `AssertionError`:
```python
>>> from punctfix import PunctFixer
>>> fixer = PunctFixer(language='da')
AssertionError: Torch not compiled with CUDA enabled
```

I realised that the reason for this is quite simple, as `device` is an argument in `PunctFixer`, which defaults to 'cuda'. To prevent others facing the same issue, you could just have a line checking if CUDA is available and set `device` to 'cpu' in that case:
```python
>>> if device == 'cuda' and not torch.cuda.is_available():
>>>     device = 'cpu'
```

--- a/punctfix/inference.py
+++ b/punctfix/inference.py
@@ -1,5 +1,6 @@
 from typing import Tuple, Dict
 
+import torch
 from transformers import TokenClassificationPipeline
 
 from punctfix.models import get_custom_model_and_tokenizer, get_english_model_and_tokenizer, \
@@ -35,7 +36,7 @@ class PunctFixer:
             self.model, self.tokenizer = get_german_model_and_tokenizer()
 
         self.model = self.model.eval()
-        self.device = -1 if device == "cpu" else 0
+        self.device = 0 if device == "cuda" and torch.cuda.is_available() else -1
 
         self.pipe = TokenClassificationPipeline(model=self.model,
                                                 tokenizer=self.tokenizer,

--- a/setup.py
+++ b/setup.py
@@ -9,7 +9,7 @@ with open('./requirements.txt', encoding="utf-8") as f:
 
 setuptools.setup(
     name="punctfix",
-    version="0.0.3",
+    version="0.0.4",
     author="Rasmus Arpe Fogh Egebæk",
     author_email="rasmus@alvenir.ai",
     description="Punctuation restoration library",

--- a/tests/test_punctuation.py
+++ b/tests/test_punctuation.py
@@ -1,4 +1,5 @@
 import unittest
+from unittest.mock import patch, MagicMock, ANY
 
 from punctfix import PunctFixer
 
@@ -72,3 +73,28 @@ class GermanPunctuationRestorationTest(unittest.TestCase):
         actual_output = self.model.punctuate(model_input)
 
         self.assertEqual(actual_output, expected_output)
+
+
+class GenerelFunctionalityTest(unittest.TestCase):
+
+    def test_if_gpu_not_available_default_cpu(self):
+        # Setup
+        self.torch_cuda_patch = patch(
+            'punctfix.inference.torch.cuda.is_available'
+        )
+        self.torch_cuda_mock: MagicMock = self.torch_cuda_patch.start()
+        self.torch_cuda_mock.return_value = False
+
+        self.token_classification_pipeline_patch = patch(
+            'punctfix.inference.TokenClassificationPipeline'
+        )
+        self.token_classification_pipeline_mock: MagicMock = self.token_classification_pipeline_patch.start()
+
+        # Test
+        self.model = PunctFixer(language="da", device="cuda")
+
+        # Verify
+        self.token_classification_pipeline_mock.assert_called_once_with(model=ANY,
+                                                                        tokenizer=ANY,
+                                                                        aggregation_strategy="first",
+                                                                        device=-1)

